yetanotherco/aligned_layer
# feat (operator): check if operator is registered on startup

- Added method to check if operator is registered with aligned layer avs on avs reader
- Used new method to check if operator is registered with avs on operator startup

To test:
1. Start anvil (`make anvil-start`)
2. Start aggregator (`make aggregator-start`)
3. Try to start operator (`make operator-start`)
4. See error
5. Register operator (`make operator-full-registration`)
6. Start operator (`make operator-start`)
